### PR TITLE
SST-1: Add a maven project (pom, checkstyle), and basic pages (readme, history, howto)

### PR DESCRIPTION
**Pull request by julianhyde**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+$ mvn compile
+```
+
+## More information
+
+* License: <a href="LICENSE">Apache Software License, Version 2.0</a>
+* Author: Julian Hyde
+* Source code: http://github.com/Stream-SQL-TCK/Stream-SQL-TCK
+* Developers list:
+  <a href="mailto:dev@calcite.apache.org">dev at calcite.apache.org</a>
```
Comment (miguno) [inline README.md:88]: Do we really want this to be done in the Calcite mailing list?  Shouldn't we rather use a separate mailing list and/or use GitHub's community features?

File: README.md [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+{% endcomment %}
+-->
+[![Build Status](https://travis-ci.org/Stream-SQL-TCK/Stream-SQL-TCK.svg?branch=master)](https://travis-ci.org/julianhyde/Stream-SQL-TCK)
+
+# Stream SQL TCK
```
Comment (miguno) [inline README.md:21]: Meta feedback: I think what's missing in the README is information on how other people can contribute and join the community around this project?  At least I would see it beneficial if community participation is actively encouraged (and guided).

File: README.md [hunk 3]
```diff
@@ -0,0 +1,73 @@
...
+
+This project is a test compatability kit (TCK) for streaming SQL.
```
Comment (miguno) [inline README.md]: Typo: compatability -> compatibility

File: src/main/config/checkstyle/checker.xml [hunk 4]
```diff
@@ -0,0 +1,285 @@
...
+  configurations.
+
+  The documentation for checkstyle is available at
+
+  http://checkstyle.sourceforge.net
+-->
+
+<!DOCTYPE module PUBLIC "-//Puppy Crawl//DTD Check Configuration 1.1//EN" "http://www.puppycrawl.com/dtds/configuration_1_1.dtd">
+
+<!-- Apache Calcite's customization of default Checkstyle behavior -->
```
Comment (arunmahadevan) [inline src/main/config/checkstyle/checker.xml]: remove reference to calcite?

File: pom.xml [hunk 5]
```diff
@@ -0,0 +1,243 @@
...
+    <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
+    <top.dir>${project.basedir}</top.dir>
+  </properties>
+
+  <!-- Environment settings. -->
+  <distributionManagement>
+    <site>
+      <id>stream-sql-tck.website</id>
+      <name>Stream SQL TCK Website</name>
+      <url>file:/home/jhyde/web2/stream-sql-tck</url>
```
Comment (guozhangwang) [inline pom.xml]: Is this intentional?

File: README.md (hunk 1 above)
Comment (julianhyde) [reply README.md:88]: None of the alternatives are perfect. Calcite is an existing list, reasonably low volume. On the call folks agreed that it was good enough.

File: README.md (hunk 2 above)
Comment (julianhyde) [reply README.md:21]: OK, I'll add that.

File: pom.xml (hunk 5 above)
Comment (julianhyde) [reply pom.xml]: Kind of intentional. Maven-release-plugin can generate a site, as long as you tell it where to put it. I'll remove the <site> tag.

**Review by guozhangwang — COMMENTED** [review]
LGTM!

**Pull request merged**